lms/services/jwt.py

```diff
@@ -51,3 +60,63 @@ def encode_with_secret(cls, payload: dict, secret, lifetime):
         jwt_str = jwt.encode(payload, secret, algorithm="HS256")
 
         return jwt_str
+
+    def decode_lti_token(self, id_token: str) -> dict:
+        """
+        Decode an LTI `id_token` JWT.
+
+        The JWT is validated against its corresponding LTIRegistration public key
+        """
+        if not id_token:
+            return {}
+
+        header = jwt.get_unverified_header(id_token)
+        if not header.get("kid"):
```

Comment:
```suggestion
        if not jwt.get_unverified_header(id_token).get("kid"):
```

What is the `header` value used for? Is this a look before you leap type of situation for something below? Is it something we could handle with a try catch instead?